tests/0.2.0.json

```diff
@@ -155,7 +152,7 @@
         "bytes": {
           "type": "integer",
           "minimum": 1,
-          "maxiumum": 9223372036854775807,
+          "maximum": 9223372036854775807,
```

Comment:
😓

Reply:
😆 ChatGPT actually caught that for me :P

Reply:
You asked gpt for a review? I should start doing that...

Reply:
Haha, nah I asked it to write some of the code :P It's pretty handy for test writing...